sdks/python/container/py310/base_image_requirements.txt

```diff
@@ -154,4 +155,5 @@ urllib3==1.26.13
 websocket-client==1.4.2
 Werkzeug==2.2.2
 wrapt==1.14.1
+xgboost==1.7.1
```

Comment:
I wouldn't put `xgboost` here since it is not a dependency of beam. 

We can have a requirements file and gradle task. We can add this gradle task to the PostCommit suites similarly on torch tests run.

For direct runner, https://github.com/apache/beam/blob/c0e689331c2a6573ecf267b9bef133a85ea8a36c/sdks/python/test-suites/direct/common.gradle#L284

For Dataflow runner,  https://github.com/apache/beam/blob/c0e689331c2a6573ecf267b9bef133a85ea8a36c/sdks/python/test-suites/dataflow/common.gradle#L403

If IT tests are light weight and easy to run, we can add them to DirectRunner.

Reply:
I followed the suggestion by @AnandInguva as well over here